### PR TITLE
Fixed JSON unmarshaling in experimental/oAuther

### PR DESCRIPTION
**Pull request by levb**
#### Summary
Fixed JSON unmarshaling in experimental/oAuther

#### Ticket Link
none

### CODE REVIEW

File: experimental/oauther/oauth2.go [hunk 1]
```diff
@@ -142,7 +142,7 @@ func (o *oAuther) GetConnectURL() string {
 
 func (o *oAuther) GetToken(userID string) (*oauth2.Token, error) {
 	var token *oauth2.Token
```
Comment (larkox) [inline experimental/oauther/oauth2.go:144]: Should we here then use the object directly instead of a pointer?

**Review by larkox — APPROVED** [review]
LGTM, just one question

File: experimental/oauther/oauth2.go (hunk 1 above)
Comment (hanzei) [reply experimental/oauther/oauth2.go:144]: Good suggestion :+1:

**Review by hanzei — APPROVED** [review]

File: experimental/oauther/oauth2.go (hunk 1 above)
Comment (levb) [reply experimental/oauther/oauth2.go:144]: It kinda doesn't matter, `json` will do the right thing, but it needs a *pointer* to the output variable

**Pull request merged**